Integrations/integration-nmap.yml

```diff
@@ -0,0 +1,98 @@
+            else:
+                tmp_host = host.address
+
+            h['Address'] = host.address
+            h['Status'] = host.status
+            svc = []
+            md += "- Nmap scan report for {0} ({1})".format(tmp_host, host.address) + '\n'
+            md += "- Host is {0}.".format(host.status) + '\n'
+            md += 'Port | State | service\n-|-|-\n'
+            for serv in host.services:
+                pserv = "{0} / {1} | {2} | {3}".format(
+                        str(serv.port),
+                        serv.protocol,
+                        serv.state,
+                        serv.service)
+                if len(serv.banner):
+                    pserv += " ({0})".format(serv.banner)
+                md += pserv
+                svc.append({'Port': serv.port, 'Protocol': serv.protocol, 'State': serv.state, 'Service': serv.service, 'Banner': serv.banner})
+            h['Services'] = svc
+            hosts.append(h)
+        scan = {'Summary': r.summary, 'Version': r.version, 'Started': r.started, 'Ended': r.endtime, 'CommandLine': r.commandline, 'ScanType': r.scan_type, 'Hosts': hosts}
+        demisto.results({'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': json.dumps(r, cls=ReportEncoder),
+            'HumanReadable': md, 'EntryContext': {'NMAP.Scan': scan}})
+  type: python
+  commands:
+  - name: nmap-scan
+    arguments:
+    - name: targets
+      required: true
+      description: The targets to scan. Accepts comma-separated list.
+    - name: options
+      required: true
+      description: The nmap options to use as documented by nmap
+    outputs:
+    - contextPath: NMAP.Scan.Summary
+      description: Scan summary
+    - contextPath: NMAP.Scan.Version
+      description: nmap version
+    - contextPath: NMAP.Scan.Started
+      description: Start time epoch
+    - contextPath: NMAP.Scan.Ended
+      description: End time epoch
+    - contextPath: NMAP.Scan.CommandLine
+      description: The command line being used
+    - contextPath: NMAP.Scan.ScanType
+      description: The type of discovery scan
+    - contextPath: NMAP.Scan.Hosts.Hostname
+      description: DNS hostname of scanned host
+    - contextPath: NMAP.Scan.Hosts.Address
+      description: Scanned host address
+    - contextPath: NMAP.Scan.Hosts.Status
+      description: Is the host up or down
+    - contextPath: NMAP.Scan.Hosts.Services.Port
+      description: The port of the service
+    - contextPath: NMAP.Scan.Hosts.Services.Protocol
+      description: The protocol of the service
+    - contextPath: NMAP.Scan.Hosts.Services.State
+      description: The state of the service
+    - contextPath: NMAP.Scan.Hosts.Services.Banner
+      description: Any captured banner from the service
+    - contextPath: NMAP.Scan.Hosts.Services.Service
+      description: The service name
+    description: Scan targets with the given parameters
+    execution: true
+  dockerimage: demisto/nmap
```

Comment:
You need to add this image to requirements.txt 
Also to the zip file we now have with all docker images

Reply:
Why requirements.txt? It's not in our standard image - it's a separate image - or maybe I'm missing something...

Reply:
You are